packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
I think ideally we want (especially) these plugins to export code in a way that the user build process can transpile them together with the user code. I was reading [this](http://2ality.com/2017/06/pkg-esnext.html) on the subject recently.